uiowa/uiowa
# Feature: uids_base superfish updates

In this PR:
Resolves: https://github.com/uiowa/uiowa/issues/1418
Work related to this story: https://github.com/uiowa/uiowa/issues/1460. 

# How to test
- Verify items on https://github.com/uiowa/uiowa/issues/1460 are complete. 
- Test all variations of the menu, toggle, horizontal nav, and mega nav at wide desktop breakpoint. 
- Test all variations of the menu, toggle, horizontal nav, and mega nav at a small mobile device breakpoint. 
- Test on ie11 to verify that it just works.
